src/octets.rs

```diff
@@ -109,28 +111,28 @@ macro_rules! put_u {
     }};
 }
 
-/// A zero-copy mutable byte buffer.
+/// A zero-copy immutable byte buffer.
 ///
 /// `Octets` wraps an in-memory buffer of bytes and provides utility functions
 /// for manipulating it. The underlying buffer is provided by the user and is
-/// not copied when creating an `Octets`. Operations are panic-free and will
+/// not copied when creating an `OctetsMut`. Operations are panic-free and will
 /// avoid indexing the buffer past its end.
```

Comment:
I think this might be an erroneous s/Octets/OctestsMut, and/or he block not reads a bit weird because it is describing an immutable byte buffer that can be manipulated.

Reply:
Yeah, copy-pasta error...